helpers/auth/decode_token.py

```diff
@@ -0,0 +1,98 @@
+import jwt
+from flask import current_app, request, jsonify, make_response
+from admin_schema import admin_schema
+from helpers.database import db_session
+from api.user.models import User
+
+class Authentication():
+  """ Authenicate token 
+    :methods
+        verify
+        decode_token
+        is_admin
+  """
+
+  def verify(self, token):
```

Comment:
Makes sense, issues is, where does the token come from ,  does not seem like it comes from the request headers?